File: ruby/two-fer/two_fer.rb
=begin
Write your code for the 'Two Fer' exercise in this file. Make the tests in
`two_fer_test.rb` pass.

To get started with TDD, see the `README.md` file in your
`ruby/two-fer` directory.
=end

class TwoFer

    def self.two_fer(name = "you")

        string = "One for #{name}, one for me."
    end
end

@class = TwoFer.two_fer("Bob")

p @class

File: zig/two-fer/two_fer.zig
const std: type = @import("std");

pub fn twoFer(buffer: []u8, name: ?[]const u8) anyerror![]u8 {
    const chosen = if (name) |n|
        if (n.len > 0) n else "you"
    else
        "you";

    return try std.fmt.bufPrint(buffer, "One for {s}, one for me.", .{chosen});
}
